Network-participating/Staking/Validator-Staking-Guide.md

```diff
@@ -162,16 +163,15 @@ There are two ways to start a node:
 # The RPC methods should be constrained to use on localhost only
 # Protect your PRC port with network security rules and the firewall
 
-docker run -p 9944:9944 -p 9933:9933 -d --restart=always \
+docker run -p 9933:9933 -d --restart=always \
 cennznet/cennznet:<VERSION> \
 --chain=/cennznet/genesis/azalea.raw.json \
 --validator \
 --name=<MY_NODE> \
 --telemetry-url 'ws://cennznet-telemetry.centrality.me:8000/submit 0' \
---rpc-methods=Unsafe \
 --ws-external \
 --rpc-external \
---rpc-cors=https://cennznet.io
+--rpc-methods=Unsafe
```

Comment:
removed the rpc cors and websocket port as it's unnecessary. by connecting to cennznet.io public full nodes validator operator can do everything they need
+ curl command shows how to get session keys via rpc